skafka/src/main/scala/com/evolutiongaming/skafka/consumer/Consumer.scala

```diff
@@ -636,6 +640,9 @@ object Consumer {
 
         val assignment = self.assignment
 
+        // TODO add metrics
```

Comment:
please create an issue in GitHub and link it here.

Reply:
done - a38c6c7ce9e8bdde1279f1c41894adecbb3ee914